### ISSUE TITLE
design a tool to check the the validity of configuration file

### ISSUE THREAD

**Issue by Harkishen-Singh** [zairza-cetb/bench-routes#88]
**Is your feature request related to a problem? Please describe.**
Currently, the user has no idea if the configuration file made by him is valid or not, depending on the standards and the fields which are required for a valid loading of the parser.

**Describe the solution you'd like**
An independent tool can be developed which has the sole purpose of checking the config file. If on erros, it can suggest the errors and the overcoming measures.

**Additional context**
The business logic _(only)_ of this tool should be integrated with the `init()` function so that the config file could be scanned right before start of the application.

**Comment by kelpikz** [comment]
hey @Harkishen-Singh, I would like to work on this issue

**Comment by muskankhedia** [comment]
@kelpikz Please go ahead.

**Comment by Tushar3099** [comment]
@muskankhedia, I would like to work on this issue if @kelpikz is not working on it.

**Comment by muskankhedia** [comment]
@kelpikz Can you please update us with the work you have done till now on this issue?

**Comment by kelpikz** [comment]
Sorry for the late reply, u can go ahead @Tushar3099

**Comment by Sanketwable** [comment]
@Tushar3099 If you are not working on this can I work on this issue?

**Comment by Tushar3099** [comment]
Yes @Sanketwable you can proceed with this issue, as I am working on other issues.
